src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md

```diff
@@ -82,6 +82,10 @@ Core](../../examples/AspNetCore/Program.cs) example app. Check this
 for configuring OpenTelemetry with `ILogger` for certain application types such
 as ASP.NET Core and .NET Worker.
 
+**ILogger Scopes**: Otlp log exporter supports exporting ILogger scopes.
+**Note**: Scope attributes with key set as empty string or `{OriginalFormat}`
+are ignored by exporter. Duplicate keys are exported as is.
+
```

Comment:
Couple comments...

* Users have to set `OpenTelemetryLoggerOptions.IncludeScopes = true` to get scopes. Might be good to mention that or link to some other doc going over the options? I found this guide but it could use some improving: https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/docs/logs/customizing-the-sdk#includescopes

* For the "note" use the github style:

   ```markdown
      > **Note**
      > Content goes here
   ```

Reply:
Updated